cgogn/geometry/algos/medial_axis.h

```diff
@@ -44,6 +45,8 @@ namespace geometry
 using geometry::Scalar;
 using geometry::Vec3;
 
+using namespace cgogn::numerics;
```

Comment:
Required for `uint32`, though the fact it compiles in spite of it would tell that the infringing template isn't instantiated